### PR TITLE
- add timestamp to distribution requests;

### PR DESCRIPTION
**Pull request by MykhailoNester**
- update unit tests;

### CODE REVIEW

File: crowdin/src/main/java/com/crowdin/platform/data/remote/api/CrowdinDistributionApi.kt [hunk 1]
```diff
@@ -5,25 +5,29 @@ import retrofit2.Call
...
-        @Path("filePath") filePath: String
+        @Path("filePath") filePath: String,
+        @Query("timestamp") timeStamp: String
     ): Call<ResponseBody>
 
     @GET("/{distributionHash}/manifest.json")
     fun getResourceManifest(
-        @Path("distributionHash") distributionHash: String
+        @Path("distributionHash") distributionHash: String,
+        @Query("timestamp") timeStamp: String
```
Comment (andrii-bodnar) [inline crowdin/src/main/java/com/crowdin/platform/data/remote/api/CrowdinDistributionApi.kt]: Not needed for manifest file, there is no caching here

File: crowdin/src/main/java/com/crowdin/platform/data/remote/api/CrowdinDistributionApi.kt [hunk 2]
```diff
@@ -5,25 +5,29 @@ import retrofit2.Call
...
     ): Call<ResponseBody>
 
     @GET("/{distributionHash}/mapping{filePath}")
     fun getMappingFile(
         @Header("if-none-match") eTag: String,
         @Path("distributionHash") distributionHash: String,
-        @Path("filePath") filePath: String
+        @Path("filePath") filePath: String,
+        @Query("timestamp") timeStamp: String
```
Comment (andrii-bodnar) [inline crowdin/src/main/java/com/crowdin/platform/data/remote/api/CrowdinDistributionApi.kt]: Not needed for mapping files, there is no caching here

File: crowdin/src/main/java/com/crowdin/platform/data/remote/api/CrowdinDistributionApi.kt (hunk 1 above)
Comment (MykhailoNester) [reply crowdin/src/main/java/com/crowdin/platform/data/remote/api/CrowdinDistributionApi.kt]: updated

File: crowdin/src/main/java/com/crowdin/platform/data/remote/api/CrowdinDistributionApi.kt (hunk 2 above)
Comment (MykhailoNester) [reply crowdin/src/main/java/com/crowdin/platform/data/remote/api/CrowdinDistributionApi.kt]: updated

**Pull request merged**